test/tests/executioners/nullspace/singular.i

```diff
@@ -0,0 +1,116 @@
+[Mesh]
+ type = GeneratedMesh
+ dim = 1
+ xmin = 0
+ xmax = 10
+ nx = 8
+[]
+
+[Problem]
+  dimNullSpace = 1
+  dimTransposeNullSpace = 1
```

Comment:
That's correct but if I don't provide it, I will have to change SteadyWithNull executioner.